spec/services/link_check_report/create_service_spec.rb

```diff
@@ -0,0 +1,124 @@
+require "spec_helper"
+
+RSpec.describe LinkCheckReport::CreateService do
+  let(:user) { FactoryGirl.create(:user) }
+
+  let(:link_check_report) { double(:link_check_report, valid?: nil, save!: nil) }
```

Comment:
Would be good to use a Factory Bot class here to make sure the fields definitely exist on the model.